source/adapters/level_zero/command_buffer.cpp

```diff
@@ -170,6 +170,64 @@ ur_result_t getEventsFromSyncPoints(
   return UR_RESULT_SUCCESS;
 }
 
+/**
+ * If necessary, it creates a signal event and appends it to the previous
+ * command list (copy or compute), to indicate when it's finished executing.
+ * @param[in] CommandType The type of the command.
+ * @param[in] CommandBuffer The CommandBuffer where the command is appended.
+ * @param[in] isFirstNode A boolean inidicating if the current node is at the
+ * beginning of the command list.
+ * @param[in] isCopy A boolean indicating if the current command uses copy
+ * engine.
+ * @param[out] ZeSignalPrevCommandEvent The event which signals when the
+ * previous command appended to a different command list is finished executing.
+ * @return UR_RESULT_SUCCESS or an error code on failure
+ */
+ur_result_t createSyncPointBetweenCopyAndCompute(
+    ur_command_t CommandType, ur_exp_command_buffer_handle_t CommandBuffer,
+    bool isFirstNode, bool isCopy,
```

Comment:
nitpicking for consistent variable naming (and update doxy comment)
```suggestion
    bool IsFirstNode, bool IsCopy,
```